pkg/controller/machineset/machineset_controller_test.go

```diff
@@ -120,21 +106,11 @@ func TestMachineSetToMachines(t *testing.T) {
 			},
 			expected: nil,
 		},
-		{
```

Comment:
Shouldn't this test case be preserved? Otherwise we aren't testing the positive test case of this method finding an existing Machine with the OwnerReference set.